ansible/files/telemetry.sh

```diff
@@ -168,11 +172,41 @@ jupyter serverextension enable --py jupyter_spark --user
 jupyter nbextension install --py jupyter_spark --user
 jupyter nbextension enable --py jupyter_spark --user
 
+# Create analyses dir whether we are mounting efs or not
+mkdir -p $HOME/analyses
+
+if [ -n "$EMAIL" ]; then
+    # Mount entire efs locally to check for user's dir 
+    mkdir -p /mnt/efs
+    sudo mount -t nfs4 -o nfsvers=4.1,rsize=1048576,wsize=1048576,hard,timeo=600,retrans=2 fs-d0c30f79.efs.us-west-2.amazonaws.com:/ /mnt/efs
+
+    if [ ! -d "/mnt/efs/$EMAIL" ]; then
+        # If no dir for this user, create it, set ownership, add examples
+        sudo mkdir -p "/mnt/efs/$EMAIL"
+
+        sudo chown hadoop:hadoop "/mnt/efs/$EMAIL"
```

Comment:
I'm not, really. I don't really *think* we're going to run into any issues like that, and since we're only mounting their specified directory, no one is accidentally going to mess up another users work. It doesn't seem to me like it's a big problem right now.